### PR TITLE
add file

### PR DESCRIPTION
**Pull request by KIGAP**

### CODE REVIEW

File: newFile.java
Comment (rasnaut) [inline newFile.java:1]: I don't like camel style of naming

**Review by rasnaut — CHANGES_REQUESTED** [review]
Please, change file according to comments

File: newFile.java
Comment (rasnaut) [reply newFile.java:1]: @ecofrau please, take look

Comment (ecofrau) [reply newFile.java:1]: @rasnaut Camel still is super dessission

**Review by rasnaut — APPROVED** [review]

**Comment by ecofrau** [comment]
i don't like your deadlin's line

**Comment by ValeryMukhin1711** [comment]
all correct

**Pull request merged**